decidim-comments/app/cells/decidim/comments/edit_comment_modal_form_cell.rb

```diff
@@ -9,6 +9,14 @@ class EditCommentModalFormCell < Decidim::ViewModel
 
       private
 
+      def cache_hash
+        hash = []
+        hash.push(I18n.locale)
+        hash.push(model.id)
+        hash.push(current_user.id)
```

Comment:
Ditto, does it make sense caching this cell with the current_user.id? Also, if it makes sense, shouldn't it be `current_user.try(:id)` as the `comment_form_cell.rb`?

Reply:
This needs to be cached. It seems that each one of the comment is adding own edit form, and this is impacting the rendering speed of the page. 
I can do the change regarding the current_user.id, but i remember the current user check is done before actually invoking this cell.